charts/loki-distributed/templates/memcached-chunks/statefulset-memcached-chunks.yaml

```diff
@@ -73,6 +73,10 @@ spec:
           {{- if .Values.memcachedChunks.persistence.enabled }}
             - --memory-file=/cache-state/memory_file
           {{- end }}
+          {{- if .Values.memcachedChunks.extstore.enabled }}
+            - -o
```

Comment:
We can't have this option here. The `-o` flag is basically an.. "open" flag. It _could_ contain ext_path like it does now, but it's open for a ton of other flags. Basically, you want that users are able to define `-o` themselves. 

I think it makes more sense to only just enable "additional PVC" and let users handle path/args themselves.